designer/client/src/actions/notificationActions.tsx

```diff
@@ -12,10 +12,11 @@ export function success(message: string): Action {
     });
 }
 
-export function error(message: string): Action {
+//TODO please take a look at this method and my changes, am I wrong or was it incomplete (without `error` and `showErrorText`) and had incomplete logic
+export function error(message: string, error?: string, showErrorText?: boolean): Action {
```

Comment:
Not sure what happens here. Why the logic is incomplete? What is the difference between message and error?

Reply:
Before my update it was like this:
```
export function error(message: string): Action {
    return Notifications.error({
        autoDismiss: 10,
        children: <Notification type={"error"} icon={<InfoOutlinedIcon />} message={message} />,
    });
}
```

but in HttpService you can find definition like this:
```
type NotificationActions = {
    ...
    error(message: string, error: string, showErrorText: boolean): void;
};
```

I was surprised that when I executed
`error("msg", "error", true)` then "msg" was shown to user. 

I've tracked it to its implementation - I've added fields I thought were missing and it worked, now you can choose if you want to show message or error, before my change I think it was impossible and flag `showErrorText` was omitted 

I might be wrong, that is why I left TODO, but for now I stand that is was bug :)

Reply:
With new implementation true/false flag selects which parameter is displayed, you can simplify to either `error("msg")` or `error("error")`. Please check usages of `HttpService.#addError` with showErrorTest = true because you are changing displayed message: raw error instead of user friendly message e.g. Failed to deploy, Failed to migrate